pkgs/servers/sql/postgresql/ext/citus.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, autoreconfHook
+, postgresql
+, curl
+, fetchFromGitHub
+, lz4
+, zstd
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "citus";
+  version = "10.0.3";
+
+  src = fetchFromGitHub {
+    owner = "citusdata";
+    repo = "citus";
+    rev = "v${version}";
+    sha256 = "hzR4tQnws5kYyLY8xzoIiOEJrCQFj0t2d+lJg2c8/AE=";
+    fetchSubmodules = false;
```

Comment:
```suggestion
```
that is the default.

Reply:
This is from `nix-prefetch-github owner repo --no-fetch-submodules --nix`, this is a bug there